apps/docs/src/docs/components/demo/PopoverCloseOnHide.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="d-flex gap-2">
+    <BPopover :click="true" :close-on-hide="true" :delay="{show: 0, hide: 0}">
+      <template #target>
+        <BButton>Click me. Popover closes when clipped</BButton>
+      </template>
+      Scroll me out of view
+    </BPopover>
+    <BPopover
+      :click="true"
+      :close-on-hide="true"
+      :delay="{show: 0, hide: 0}"
+      :boundary-padding="{top: navHeight}"
+    >
```

Comment:
```suggestion
    <BPopover :click="true" :delay="{show: 0, hide: 0}" :boundary-padding="{top: navHeight}">
```

I think this is supposed not to close